DigitalLearningSolutions.Data.Tests/Services/UserServiceTests.cs

```diff
@@ -1093,61 +1093,6 @@ public void UpdateUserAccountDetailsViaDelegateAccount_calls_UpdateDelegateProfe
             ).MustHaveHappened();
         }
 
-        [Test]
-        public void UpdateUserAccountDetailsViaDelegateAccount_updates_single_account_if_no_email_set()

```

Comment:
I think we still want this test in, but modify the assertions to check that the call to update multiple delegates only has an empty array of ids passed in.